docs/learn/learn-async-backing.md

```diff
@@ -0,0 +1,146 @@
+block, as the parablock anchors itself to it). This means that each parablock is limited to the
+window of the relay chain's block time.
+
+:::info
+
+It is important to note two following clarifications:
+
+1. The parablock is itself within the relay chain, but rather the **paraheader**
+2. The relay chain does not validate the entire state of a parachain, but only the state transitions
+   within that block.
+
+:::
+
+For more information on the validity and availability process, be sure to visit the
+[parachain protocol](../learn/learn-parachains-protocol.md) page.
+
+## Synchronous Backing on Polkadot
+
+With synchronous backing, there was only about a single relay chain block, or 6-second window, to
+complete the parablock inclusion process. This was tightly coupled to the relay chain's progress,
+where blocks had to be created within this window:
+
+```mermaid
+%%{init: { 'logLevel': 'debug', 'theme': 'neutral', 'themeVariables': { 'fontSize': '14px', 'commitLabelFontSize': '16px', 'tagLabelFontSize': '16px' }, 'gitGraph': {'showBranches': true, 'showCommitLabel':true,'mainBranchName': 'Relay Chain'}} }%%
+gitGraph
+commit id:"R1"
+branch "Parachain 1, Block 1" order: 2
+commit id:"Block 1 (Candidate)"
+commit id:"Block 1 (Backed)"
+checkout "Parachain 1, Block 1"
+commit id:"Block 1 (Included)"
+branch "Parachain 1, Block 2" order: 3
+checkout "Relay Chain"
+merge "Parachain 1, Block 1" tag:"P1 Included" id:"R2"
+checkout "Parachain 1, Block 2"
+commit id:"Block 2 (Candidate)"
+commit id:"Block 2 (Backed)"
+checkout "Parachain 1, Block 2"
+commit id:"Block 2 (Included)"
+checkout "Relay Chain"
+merge "Parachain 1, Block 2" tag:"P2 Included" id:"R3"
+```
+
+where **P1** and **P2** are two parablocks, and **R1** and **R2** are two relay-chain blocks. The
+letters within the parentheses represent the state of a parachain block. Hence **C** (candidate,
+created by collators), **B** (backed, by para-validators), and **I** (included, by block authors in
+the relay-chain). For more information about parablock validation see the
+[parachain protocol page](./learn-parachains-protocol.md).
+
+In the diagram there are three relay-chain blocks being finalized (**R1-2**) and a parachain
+(**Parachain 1**) proposing two parablocks (**P1** and **P2**). Each parablock's lifecycle, from
+being candidate (**P(C)**) to being backed and included (**P(B)** and **P(I)**, respectively), must
+fit within the relay-chain block lifetime.
+
+Thus, the main limitation of synchronous backing is that parablock validation is tightly coupled to
+the relay chain's progression on a 1-1 basis, meaning every parablock must be built, backed, and
+included within six seconds. Parablocks depend on being in sync with the relay chain, reducing the
+amount of data in the block.
+
+Essentially, a parablock is rushing to being validated by the relay-chain due to this synchrony.
+
+By making this process of backing parablocks more asynchronous, parachains get the chance to not
```

Comment:
The point about including more data in each parablock was covered sufficiently only a few lines earlier IMO. Instead I'd dive deeper into how Async Backing allows us to retry the availability process.

Here's a short description: Previously a parablock produced with relay parent `N` would only be valid for backing at relay block `N + 1` and for inclusion at relay block `N + 2`. 

With asynchronous backing there is a configuration `max_ancestry_len` which determines how long a parablock remains usable for. For instance, with `max_ancestry_len = 6` a parablock with relay parent `N` could be backed at `N + 6` at the latest and included at `N + 7`. This gives us room for 5 potential availability retries!

Reply:
This definitely warrants a diagram to best showcase this I'm thinking

Reply:
A good thought. I haven't made a diagram for this either. By the way, here's a link to the visuals I created for my async backing PBA presentation. Rob was in attendance and didn't spotlight any errors, so I think these are pretty good to use. Feel free to make use of them! [link](https://docs.google.com/presentation/d/1AQNKRV1C26aJfQFRscrWt_cvIiRdXQu26SWP0B7Y80Y/edit?usp=sharing)